### DISCUSSION TITLE
Livewire Session Token

### DISCUSSION THREAD

**Discussion by bilfeldt** [osiset/laravel-shopify#1253] [Q&A]
Has anyone successfully implemented Livewire for en embedded app using the [Session Token](https://shopify.dev/apps/auth/oauth/session-tokens) authentication?

This package has a config described like so:

> If you use Turbolinks/Turbo and Livewire, turn on this setting to get the token assigned automatically.

It's unclear for me if [Turbo (by Hotwire)](https://github.com/hotwired/turbo) (or the deprecated [Turbolink](https://github.com/turbolinks/turbolinks)) is really needed for Livewire to function or if it's enough somehow to [hook into Livewire](https://laravel-livewire.com/docs/2.x/lifecycle-hooks#js-hooks) for finding a new session token and adding that as Bearer token on the Livewire AJAX request.

**Comment by bilfeldt** [comment]
Seems to have been address by @unplugged216 in https://github.com/osiset/laravel-shopify/pull/1247 👍

**Comment by Kyon147** [comment]
This has been released with `v17.3.2` today.

**Reply by Kyon147** [reply]
https://github.com/osiset/laravel-shopify/releases/tag/v17.3.2
